lib/fluent/plugin/in_tail.rb

```diff
@@ -370,7 +370,7 @@ def refresh_watchers
       target_paths_hash = expand_paths
       existence_paths_hash = existence_path
 
-      log.debug { "tailing paths: target = #{target_paths.join(",")} | existing = #{existence_paths.join(",")}" }
+      log.debug { "tailing paths: target = #{target_paths_hash.keys.join(",")} | existing = #{existence_paths_hash.keys.join(",")}" }
```

Comment:
The keys of `target_paths_hash` and `existence_paths_hash` are inode when `follow_inode = true`.
So that paths should be collected by `xxx_hash.collect { |key, target_info| target_info.path }`.

Reply:
Done and tested ok with `follow_inodes = true`